libs/runtime/ebpf_ring_buffer.c

```diff
@@ -187,7 +205,7 @@ ebpf_ring_buffer_return(_Inout_ ebpf_ring_buffer_t* ring, size_t length)
 
 Done:
     ebpf_lock_unlock(&ring->lock, state);
-    return result;
+    EBPF_RETURN_RESULT(result);
```

Comment:
ebpf_async.c etc always do EBPF_LOG_ENTRY() in all functions that do EBPF_RETURN_RESULT().  Shouldn't this be consistent?

Reply:
Fixed.